### PR TITLE
Maven initialize project OVG. Add plugins on pom.xml.

### PR DESCRIPTION
**Pull request by ListerKred**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+  <version>1.0-SNAPSHOT</version>
+
+  <name>OVG</name>
+  <!-- FIXME change it to the project's website -->
+  <url>http://www.example.com</url>
+
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <maven.compiler.source>1.7</maven.compiler.source>
+    <maven.compiler.target>1.7</maven.compiler.target>
```
Comment (Pamwamba) [inline pom.xml]: Set java version to 1.8

File: src/main/java/com/ovg/app/App.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
+package com.ovg.app;
+
+/**
+ * Hello world!
+ *
+ */
+public class App 
+{
```
Comment (Pamwamba) [inline src/main/java/com/ovg/app/App.java]: Put the bracket at the end of the line

File: src/main/java/com/ovg/app/App.java [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+    public static void main( String[] args )
+    {
```
Comment (Pamwamba) [inline src/main/java/com/ovg/app/App.java]: Put the bracket at the end of the line

File: src/test/java/com/ovg/app/AppTest.java [hunk 4]
```diff
@@ -0,0 +1,13 @@
+package com.ovg.app;
+
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Test;
+
+public class AppTest {
+
+    public void shouldAnswerWithTrue()
+    {
```
Comment (Pamwamba) [inline src/test/java/com/ovg/app/AppTest.java]: Put the bracket at the end of the line

**Comment by ListerKred** [comment]
It's done.

**Pull request merged**